agent/virustotal.py

```diff
@@ -16,16 +15,15 @@ class VirusTotalApiError(Error):
     """VirtualTotalApiError."""
 
 
-def scan_file_from_message(message: msg.Message, api_key: str) -> dict:
+def scan_file_from_message(file_content: bytes, api_key: str) -> dict:
     """Method responsible for scanning a file through the Virus Total public API.
     Args:
-        message: Message containing the file to scan.
+        file_content: Message containing the file to scan.
         api_key : Key for the virustotal api.
     Returns:
         response: The response of the Virus Total public API.
     """
-    file = message.data["content"]
-    file_md5_hash = hashlib.md5(file)
+    file_md5_hash = hashlib.md5(file_content)
     hash_hexa = file_md5_hash.hexdigest()
```

Comment:
```suggestion
    file_hash = hashlib.md5(file_content).hexdigest()
```

Reply:
@BlueSquare1 Please stick with the part of the code that is affected by the PR.